client/my-sites/checkout/checkout-thank-you/pending/index.tsx

```diff
@@ -37,6 +37,7 @@ interface CheckoutPendingProps {
 	receiptId: number | undefined;
 	siteSlug?: string;
 	redirectTo?: string;
+	fromSite?: string;
```

Comment:
Could you add a comment above this definition (and all the other places `fromSite`/`fromSiteSlug` is used) to explain what it means? I think that "fromSite" is ambiguous enough that it would benefit from having an explanation so it doesn't get misused in the future.

```suggestion
    /**
    * I don't know what this means...
    */
	fromSite?: string;
```

Reply:
Yes, totally, good idea.  Thanks!

Reply:
✅